src/components/RecentLinks.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Fragment } from 'react';
+
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+
+import { graphql } from 'gatsby';
+
+export default ({ links, showAmmount }) => {
```

Comment:
A small typo here - amount should only have one "m"

Reply:
Done :)